src/main/java/noppes/npcs/client/gui/SubGuiNpcRespawn.java

```diff
@@ -55,6 +55,7 @@ protected void actionPerformed(GuiButton guibutton){
 		}
 		else if(button.id == 1){
 			stats.canDespawn = button.getValue() == 1;
+			stats.playerSetCanDespawn = stats.canDespawn;
```

Comment:
View comment on DataStats, can't figure out what ``playerSetCanDespawn`` should do.